src/sentry/runner/commands/createuser.py

```diff
@@ -49,7 +47,16 @@ def _set_superadmin(user):
 
 
 @click.command()
-@click.option("--email")
+@click.option(
+    "--email",
+    default=None,
+    help="Email to create account for, accepts a comma separated list of emails if multiple accounts are to be created.",
```

Comment:
Click also has the [`Multiple=True` option](https://click.palletsprojects.com/en/8.1.x/options/#multiple-options) which would let you do `--email xxx@foo.com --email yyy@foo.com` and give you an array out of it. Not sure if that's better or worse to you. :)

Reply:
One more small readability thing (but not sure if it would work - do click option names and args have to match or is it just the order you define them in?):

`def createuser(email,..)` could become `emails` and then you could do `for email in emails:`

Reply:
Click by default names these, but verified we can pass in a custom variable name to the option.